.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .idea/
+index2.php
```

Comment:
why?

Reply:
sorry was testing something and it came up i should just delete the file haha